echopype/visualize/api.py

```diff
@@ -171,41 +171,41 @@ def create_echogram(
     )
 
 
-def _add_water_level(
+def _add_range_offset(
     range_in_meter: xr.DataArray,
-    water_level: Union[int, float, xr.DataArray, bool],
+    range_offset: Union[int, float, xr.DataArray, bool],
     data_type: str,
     platform_data: Optional[xr.Dataset] = None,
 ) -> xr.DataArray:
-    if isinstance(water_level, bool):
-        if water_level is True:
+    if isinstance(range_offset, bool):
+        if range_offset is True:
             if data_type == xr.Dataset:
                 warnings.warn(
-                    "Boolean type found for water level. Ignored since data is an xarray dataset."
+                    "Boolean type found for range offset. Ignored since data is an xarray dataset."
```

Comment:
Seems like `Range offset` should also be `range_offset` here since it is a variable name.
Please also check others in this script. Thanks!